conf/quincy/baremetal_pipeline/mero_conf.yaml

```diff
@@ -95,13 +90,11 @@ globals:
             - /dev/sdj
             - /dev/sdk
             - /dev/sdl
-            - /dev/nvme0n1
         - hostname: mero011
```

Comment:
Can we have same no of disks across all the OSD nodes? We need to have a homogenous cluster for testing.

This is the current distribution : 
mero001 (node1): No volumes listed.
mero003 (node2): No volumes listed.
mero004 (node3): No volumes listed.
mero005 (node4): 11 volumes listed.
mero007 (node5): 12 volumes listed.
mero009 (node6): 12 volumes listed.
mero011 (node7): 11 volumes listed.
mero013 (node8): 11 volumes listed.
mero014 (node9): 12 volumes listed.
mero015 (node10): No volumes listed.
mero016 (node11): No volumes listed.
mero017 (node12): 12 volumes listed.
mero018 (node13): 12 volumes listed.
mero019 (node14): 12 volumes listed.
mero020 (node15): 12 volumes listed.

Reply:
Hi Pawan,
While reimaging, we've noticed that device labels are being assigned randomly. 
To maintain uniformity in our cluster, we calculate the OSD (Object Storage Daemon) devices separately when applying the OSD configuration in a different script.
Can you please verify if the roles within the stretched clusters are evenly balanced?

Regards,
Amarnath